test/datahike/test/api_test.cljc

```diff
@@ -764,16 +765,28 @@
                                                                   5          2
                                                                   6          3}})))))
         update-for-history
-        (fn [metrics schema-on-write?]
+        (fn [metrics schema-on-write? attribute-refs?]
           (->> (update metrics :count #(+ % 4))
                (merge-with merge {:per-attr-counts     {:db/txInstant 4}
-                                  :per-entity-counts   {(+ tx0 1)  1
-                                                        (+ tx0 2)  1
-                                                        (+ tx0 3)  1
-                                                        (+ tx0 4)  1}
+                                  :per-entity-counts   {(+ tx0 1) 1
+                                                        (+ tx0 2) 1
+                                                        (+ tx0 3) 1
+                                                        (+ tx0 4) 1}
                                   ; 10 == 11 minus 1 parent datom that wouldn't get added unless retracted
-                                  :temporal-count      (+ 11 (if schema-on-write? 10 0))
-                                  :temporal-avet-count (if schema-on-write? 9 0)})))
+                                  :temporal-count      (+ 11 (if schema-on-write?
+                                                               (if attribute-refs?
+                                                                 79
+                                                                 10)
+                                                               (if attribute-refs?
+                                                                 79
+                                                                 0)))
+                                  :temporal-avet-count (if schema-on-write?
+                                                         (if attribute-refs?
+                                                           48
+                                                           9)
+                                                         (if attribute-refs?
+                                                           48
+                                                           0))})))
```

Comment:
Can you name the magical numbers in this function, so we don't have to start rethinking what was meant by them? Alternatively, you could write how they come about in comments.

Reply:
Thanks for fixing this.